models/test/mapper/TestPreviewMapper.php

```diff
@@ -230,11 +231,16 @@ private function getService(): Service
 
     /**
      * @param $categories
-     *
+     * @param $itemRef
      * @return bool
      */
-    private function isItemInformational($categories): bool
+    private function isItemInformational($categories, $itemRef): bool
     {
-        return in_array('x-tao-itemusage-informational', $categories, true);
+        $additionalCheck = false;
+
+        if ($itemRef instanceof ExtendedAssessmentItemRef && !count($itemRef->getResponseDeclarations())){
```

Comment:
the same as in the test runner: https://github.com/oat-sa/extension-tao-testqti/blob/master/helpers/class.TestRunnerUtils.php#L734